game2.py

```diff
@@ -0,0 +1,108 @@
+
+import pygame
+from pygame.locals import *
+import math
+import random
+
+# Initialize the game
+pygame.init()
+width, height = 1200, 700
+screen=pygame.display.set_mode((width, height))
+keys = [False, False, False, False]
+playerpos=[600, 680]
+
+
+
+#  this is a timer for the badguys
+badtimer=100
+badtimer1=0
+badguys=[[640,100]]
+bad_list = []
+
+
+# 3 - Load images
+player = pygame.image.load("basketball.png")
+player = pygame.transform.scale(player, (100, 100))
+grass = pygame.image.load('resources/images/grass.png')
+goal = pygame.image.load('goal.png')
+goal = pygame.transform.scale(goal, (200, 200))
+court = pygame.image.load('halfcourt.jpg')
+court = pygame.transform.scale(court, (1200, 700))
+badguyimg1 = pygame.image.load("bad_guy.png")
+badguyimg1 = pygame.transform.scale(badguyimg1, (100, 100))
+
+#these images don't loop through?
+# badguyimg2 = pygame.image.load("bad_guy2.gif")
+# badguyimg2 = pygame.transform.scale(badguyimg2, (100, 100))
+# badguyimg3 = pygame.image.load("bad_guy3.gif")
+# badguyimg3 = pygame.transform.scale(badguyimg3, (100, 100))
+# badlist = [badguyimg1, badguyimg2, badguyimg3]
+
+
+#  Game Logic
+while 1:
+    badtimer -= 1
+    screen.blit(court, (0, 0))
+    screen.blit(goal,(487, 0))
+    
+    #This sets the initital positions
+    position = pygame.mouse.get_pos()
+    angle = math.atan2(position[1]-(playerpos[1]+32),position[0]-(playerpos[0]+26))
+    playerrot = pygame.transform.rotate(player, 360-angle*57.29)
```

Comment:
Adding quick comments to the individual lines may help readability. e.g. I'm guessing the "playerrot" var sets the rotation, but I'm not sure.

Reply:
yes it does!
Thank you!